src/Microsoft.FeatureManagement/Microsoft.FeatureManagement.csproj

```diff
@@ -22,22 +22,31 @@
     <Description>Microsoft.FeatureManagement provides a way to develop and expose application functionality based on features. Many applications have special requirements when a new feature is developed such as when the feature should be enabled and under what conditions. This library provides a way to define these relationships, and also integrates into common .NET code patterns to make exposing these features possible.</Description>
     <Authors>Microsoft</Authors>
     <Company>Microsoft</Company>
-    <PackageProjectUrl>https://github.com/Azure/AppConfiguration</PackageProjectUrl>
-    <PackageLicenseUrl>https://licenses.nuget.org/MIT</PackageLicenseUrl>
```

Comment:
https://learn.microsoft.com/en-us/nuget/reference/nuspec#licenseurl 
PackageLicenseUrl is also deprecated. Thank you for calling out it.

Reply:
No problem & also expression's are a lot more nicely integrated into dependency analysis.